python/cudf/cudf/core/index.py

```diff
@@ -434,21 +459,34 @@ def to_pandas(self):
             name=self.name,
         )
 
-    @property
+    @property  # type: ignore
```

Comment:
Why does this require a type ignore? We're not applying it after annotate in this case.

Reply:
yes, this was an unnecessary find-replace.